src/app/game-state/battle.service.ts

```diff
@@ -144,7 +144,11 @@ export class BattleService {
       for (let i = 0; i < enemyStack.quantity; i++){
         if (Math.random() < enemyStack.enemy.accuracy){
           let damage = enemyStack.enemy.attack;
-          damage = damage / (1 + this.characterService.characterState.defense * 0.1);
+          let combatDefense = this.characterService.characterState.defense;
+          // with 20k max health currently, the curve has to be related to Death's damage so player has the chance to survive and oneshot Death.
+          if (combatDefense > 0) damage = damage / (1 + Math.pow(100 + this.enemyRepo.death.attack / 20000, (-damage + combatDefense) / combatDefense) );
```

Comment:
Nitpick: bracket all if statements. I know one-liners are legal, but it's the way all the rest of the codebase is and I'd like to keep it consistent for better readability.